clusterloader2/pkg/util/cluster.go

```diff
@@ -38,9 +45,22 @@ func GetSchedulableUntainedNodes(c clientset.Interface) ([]corev1.Node, error) {
 	if err != nil {
 		return nil, err
 	}
+	prometheusPods, err := client.ListPodsWithOptions(c, monitoringNamespace, metav1.ListOptions{
```

Comment:
I'm sorry, I didn't realize that before.
This logic shouldn't go to GetUnschedulableUntaintedNodes - this is very generic function that is used in many context.

This logic should go directly to chaos monkey code here:
https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/pkg/chaos/nodes.go#L67

Reply:
+1

Reply:
@mm4tt @mm4tt Have fixed!  Sorry for not noticing this